test/intl402/Temporal/PlainYearMonth/prototype/monthCode/chinese-calendar-dates.js

```diff
@@ -18,7 +18,7 @@ const leapMonthCases = [
   { year: 1982, month: 5, monthCode: "M04L", day: 1, referenceYear: 1963, referenceDay: 23 },
   // See https://github.com/tc39/proposal-intl-era-monthcode/issues/60
   // { year: 1984, month: 11, monthCode: "M10L", day: 1, referenceYear: 1870, referenceDay: 23 },
-  { year: 1987, month: 7, monthCode: "M06L", day: 1, referenceYear: 1960, referenceDay: 24 },
+  { year: 1987, month: 7, monthCode: "M06L", day: 1, referenceYear: 1960, referenceDay: 26 },
```

Comment:
oh, good catch - I guess this is not used in the other files where it's specified as 24 and that's why those tests didn't fail